grafana_dashboards/components/panels.py

```diff
@@ -29,21 +29,28 @@ def __init__(self, data, registry):
 
 
 class PanelsItemBase(JsonGenerator):
-    pass
+    _default_span = None
 
+    def copy_basic_fields(self, source, dest):
+        for field in {'description', 'transparent', 'repeat'}:
+            if field in source:
+                dest[field] = source[field]
+
+        dest['span'] = source.get('span', self._default_span)
+        dest['title'] = source.get('title', None)
 
-class Graph(PanelsItemBase):
 
+class Graph(PanelsItemBase):
+    _default_span = 12
     _copy_fields = {'stack', 'fill', 'aliasColors', 'leftYAxisLabel', 'bars', 'lines', 'linewidth', 'y_formats',
                     'x-axis', 'y-axis', 'points', 'pointradius', 'percentage', 'steppedLine', 'repeat',
                     'minSpan', 'datasource'}
 
     def gen_json_from_data(self, data, context):
         panel_json = super(Graph, self).gen_json_from_data(data, context)
+        self.copy_basic_fields(data, panel_json)
```

Comment:
Please instead of defining new method I think it would be better to rely on inheritance and just define method `gen_json_from_data` in `PanelsItemBase` and handle common properties there.